text/0005-sapper-as-middleware.md

```diff
@@ -0,0 +1,165 @@
+- Start Date: 2020-08-31
+- RFC PR: 
+- Svelte Issue: 
+
+# Sapper as Middleware
+
+## Summary
+
+The Sapper build process should emit a mostly self-contained blob of javascript (and supporting files) that
+implements all server-side routes, SSR, and DOM code for the entire application. This javascript should
+export (at least) two functions:
+
+- A `middleware(req, res)` function that will serve a dynamic request.
+- A `assets(req, res)` function that will serve a static asset request.
+
+The supporting files *must* also include a manifest of files that can be uploaded to a CDN.
+
+This javascript *must not* start an Express server.
+
+## Motivation
+
+Working with Sapper in production for a large scale project has identified a number of
+challenges that arise from the unintended complexity of stale assumptions.
+
+By reducing these assumptions to the *absolute minumum*, we aim to:
+
+- Make it easier to deploy Sapper in efficient, scalable configurations
+- Make it easy to use Sapper applications in more scenarios than just a single standalone site
+- Make it easier to extend Sapper to suit specific project needs (without requiring modifications of Sapper itself)
+
+As of 0.28, the Sapper build process emits a completely standalone directory in `__sapper__/build`
```

Comment:
I may have missed what changed in 0.28, do you have a link to the relevant PR that changed the build output?

Reply:
No, this is bad wording. I'm trying to say that it's true in the current version and anchor it to a specific point in time so that someone looking back can know what "current" referring to.